sql/core/src/test/resources/sql-tests/inputs/udf/pgSQL/udf-select_having.sql

```diff
@@ -6,7 +6,7 @@
 -- https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/sql/select_having.sql
 --
 -- This test file was converted from inputs/pgSQL/select_having.sql
--- TODO: We should add UDFs in GROUP BY clause when [SPARK-28445] is resolved.
+-- TODO: We should add UDFs in GROUP BY clause when [SPARK-28386] and [SPARK-26741] is resolved.
```

Comment:
I am a bit lost about this or I forget something.
Can't we add UDF in group-by clause (resolved in SPARK-28445)?

Reply:
@HyukjinKwon 
From this 
https://github.com/apache/spark/pull/25215#issuecomment-518897609
I thought I need to update todo with blocking jira numbers

Reply:
Ah, right. I forgot. Can we enable all other tests with UDF in group-by and comment out the test?

```
-- !query 11
SELECT udf(b), udf(c) FROM test_having
GROUP BY udf(b), udf(c) HAVING udf(count(*)) = 1 ORDER BY udf(b), udf(c)
-- !query 11 schema
struct<>
-- !query 11 output
org.apache.spark.sql.AnalysisException
cannot resolve 'b' given input columns: [CAST(udf(cast(b as string)) AS INT), CAST(udf(cast(c as string)) AS STRING)]; line 2 pos 63
```

I guess we can still add some more tests?